resources/views/manager/builder-06.html.twig

```diff
@@ -501,7 +501,7 @@
                 <h5 data-c-font-weight="bold" data-c-margin="top(double) bottom(half)">
                     Can't find the skill you need?
                 </h5>
-                <p data-c-margin="bottom(normal)">Building a skills list is a huge endeavour, and it's not surprising that Talent Cloud's list doesn't have the skill you're looking for. To help us expand our skill list, please <a href="mailto:talent.cloud-nuage.de.talents@tbs-sct.gc.ca" title="Get in touch with Talent Cloud to have a skill added to the platform.">get in touch with us through email</a>. Provide the skill's name, as well as a short description to kick-off the discussion.</p>
+                <p data-c-margin="bottom(normal)">Building a skills list is a huge endeavour, and it's not surprising that Talent Cloud's list doesn't have the skill you're looking for. To help us expand our skill list, please <a href="mailto:{{ config('mail.admin_address') }}" title="Get in touch with Talent Cloud to have a skill added to the platform.">get in touch with us through email</a>. Provide the skill's name, as well as a short description to kick-off the discussion.</p>
```

Comment:
I believe this file and builder-01 are just the UI mock up Josh built for the JPB haha. I guess we can delete these files now. I'll make a issue for it.